composer.json

```diff
@@ -15,7 +15,7 @@
         "nikic/php-parser": "~4.12.0",
         "phpstan/phpdoc-parser": "^0.5.5",
         "sabre/xml": "~2.2.3",
-        "symfony/console": "~4.1.0||~4.4.0",
+        "symfony/console": "~4.1.0||~4.4.0||~5.4.0",
```

Comment:
@anzin Can we drop 4.1.x ?

Reply:
Hello @xmav, I dropped 4.1.x thank you for review.